less/common/Checkbox.less

```diff
@@ -31,6 +31,10 @@
   background: @control-bg;
   .transition(background-color 0.2s);
 
+  .Modal-content & {
+    background: @body-bg;
+  }
+
```

Comment:
For consistency, this should be added to the Modal.less file.It should also be applied to `.Modal-body` since that's the only part with the `@control-bg` background.

Reply:
Thanks @SychO9, agree on both points. Have made the modifications - please have a look. Note that due to the way LESS appends or prepends parent selectors the added styles in Modal.less have a bit more verbose than essentially a one-liner in Checkbox.less.

Reply:
Thank you, I think we might be able to use a one liner it we used the `off` class, can you please test if that's the case ?